rtt/transports/corba/RemotePorts.cpp

```diff
@@ -237,6 +245,25 @@ DataSourceBase::shared_ptr RemoteOutputPort::getDataSource() const
     return DataSourceBase::shared_ptr();
 }
 
+
+bool RemoteOutputPort::disconnect(PortInterface* port)
+{
+    RemoteInputPort *portI = dynamic_cast<RemoteInputPort *>(port);
+
+    //if not a remote port, we can not handle at the moment!
+    if(portI == NULL){
+        Logger::In in("RemoteOutputPort::disconnect(PortInterface& port)");
+        log(Error) << "Port: " << port->getName() << " could not be disconnected from: " << this->getName()
```

Comment:
missing space or `nlog()` after port name:
```
1.230 [ ERROR  ][RemoteOutputPort::disconnect(PortInterface& port)] Port: mi could not be disconnected from: mobecause it could not be casted to an RemoteInputPort type!
Only disconnect of two remote ports supported by corba layer, yet!

```

RTT Logger has a `nlog()` method (see documentation [here](http://orocos-toolchain.github.io/devel/documentation/rtt/master/api/html/classRTT_1_1Logger.html#ac6fabc2ad9f34905a576dff32952e173)), which seems to be preferred over `std::endl`.